docs/sources/administration/metrics.md

```diff
@@ -8,8 +8,95 @@ parent = "admin"
 weight = 8
 +++
 
-# Internal metrics
+# Internal Grafana metrics
 
-Grafana collects some metrics about itself internally. Currently, Grafana supports pushing metrics to Graphite or exposing them to be scraped by Prometheus.
+Grafana collects some metrics about itself internally. Grafana supports pushing metrics to Graphite or exposing them to be scraped by Prometheus.
 
-To emit internal metrics you have to enable the option under the [metrics] section in your [grafana.ini](http://docs.grafana.org/installation/configuration/#enabled-6) config file. If you want to push metrics to Graphite, you must also configure the [metrics.graphite](http://docs.grafana.org/installation/configuration/#metrics-graphite) section.
+For more information about configuration options related to Grafana metrics, refer to [metrics]({{< relref "../installation/configuration/#metrics" >}}) and [metrics.graphite]({{< relref "../installation/configuration/#metrics-graphite" >}}) in [Configuration]({{< relref "../installation/configuration.md" >}}).
+
+## Available metrics
+
+When enabled, Grafana exposes a number of metrics, including:
+
+* Active Grafana instances
+* Number of dashboards, users, and playlists
+* http status codes
+* Requests by routing group
+* Grafana active alerts
+* Prometheus alerts
+* Grafana performance
+
+## View Grafana metrics in Prometheus
+
+These instructions assume you have already added Prometheus as a data source in Grafana.
+
+1. Enable sending metrics to Prometheus. In your configuration file (`grafana.ini` or `custom.ini` depending on your operating system) remove the semicolon to enable the following configuration options: 
+
+   ```
+   # Metrics available at HTTP API Url /metrics
+   [metrics]
+   # Disable / Enable internal metrics
+   enabled           = true
+
+   # Disable total stats (stat_totals_*) metrics to be generated
+   disable_total_stats = false
+   ```
+
+2. (optional) If you want to require authorization to view the metrics endpoint, then uncomment and set the following options:
+
+   ```
+   basic_auth_username =
+   basic_auth_password =
+   ```
+
+3. Restart Grafana. Grafana now sends metrics to http://localhost:3000/metrics.
+4. Add the job to your prometheus.yml file.
+   Example:
+
+   ```
+   - job_name: 'grafana_metrics'
+
+      scrape_interval: 6s
```

Comment:
```suggestion
      scrape_interval: 15s
```

15s is the default so let's stick to that.
Kudos if you picked 6s because its a tenth of 60s :) But it's **very** uncommon to use that.

Reply:
Actually ended up with that because it was in a Getting started tutorial.